python/build.gradle

```diff
@@ -1,8 +1,82 @@
+import com.adyen.sdk.Service
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+
 plugins {
     id 'adyen.sdk-automation-conventions'
 }
 
 project.ext {
-    // @TODO paths from config file don't work
-//    configFile = 'templates/config.yaml'
+    generator = 'python'
+    removeTags = false
+}
+
+List<Service> services = project.ext.services
+services.find { it.id == 'payment' }.small = false
+
+// Service renaming
+Map<String, String> serviceNaming = project.ext.serviceNamingCamel
+serviceNaming.putAll([
+        'binlookup'            : 'binlookup',
```

Comment:
Both are called `binlookup`. What is being mapped here?

Reply:
The key it's the service ID and the value it's new name. The original would have a different case, it's [BinLookup](https://github.com/Adyen/adyen-dotnet-api-library/tree/main/Adyen/Model/BinLookup).